app/helpers/miq_ae_class_helper.rb

```diff
@@ -324,4 +324,26 @@ def datastore_form(ae_ns, sb_data, type)
                             :nameReadOnly  => domain && !ae_ns.editable_property?(:name),
                             :descReadOnly  => domain && !ae_ns.editable_property?(:description)})
   end
+
+  def first_user_summary(ae_method, sb)
```

Comment:
please change the `sb` to `sb_data` or some meaning fun name

Reply:
Made the changes